src/js/components/Skeleton/stories/CustomThemed/Custom.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import { Box, Button, Heading, Grommet, Text } from 'grommet';
+
+export const Custom = () => {
+  const customTheme = {
+    skeleton: {
+      colors: {
+        light: ['#a2a8a8', '#adb9ba'],
+      },
+      round: 'xsmall',
```

Comment:
I'm not seeing the rounding:
<img width="1329" alt="Screen Shot 2022-11-09 at 11 21 29 AM" src="https://user-images.githubusercontent.com/12522275/200921999-f9f1c0a3-15b6-4e53-8f92-8f48c91b898b.png">